<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TestPluginDialogBase</class>
 <widget class="QDialog" name="TestPluginDialogBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1108</width>
    <height>594</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Test Plugin</string>
  </property>
  <widget class="QLabel" name="image_label">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>40</y>
     <width>600</width>
     <height>492</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="info_label">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>80</y>
     <width>461</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QComboBox" name="image_type">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>400</y>
     <width>261</width>
     <height>45</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Карта изменений</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Сегментированное изображение</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Оригинальное изображение</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="export_button">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>520</y>
     <width>261</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string>Экспортировать данные</string>
   </property>
  </widget>
  <widget class="QComboBox" name="data_format">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>460</y>
     <width>261</width>
     <height>45</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>AVRO</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>CSV</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>JSON</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>POSTGRESQL</string>
    </property>
   </item>
  </widget>
  <widget class="QProgressBar" name="progress_bar">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>20</y>
     <width>211</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QScrollArea" name="scroll_area">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>771</width>
     <height>581</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>769</width>
      <height>579</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QSpinBox" name="image_size_selector">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>190</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>200</number>
   </property>
   <property name="value">
    <number>5</number>
   </property>
  </widget>
  <widget class="QPushButton" name="change_size_button">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>230</y>
     <width>121</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Изменить размер</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
